.github/workflows/release-go-task.yml

```
@@ -132,14 +147,11 @@ jobs:
         run: |
           gon "${{ env.GON_CONFIG_PATH }}"
 
-      - name: Re-package binary and output checksum
+      - name: Re-package binary
         id: re-package
         working-directory: ${{ env.DIST_DIR }}
         # This step performs the following:
         # 1. Repackage the signed binary replaced in place by Gon (ignoring the output zip file)
```

```suggestion
        # Repackage the signed binary replaced in place by Gon (ignoring the output zip file)
```
It's useless to have a numbered list with only a number

---

This also applies to https://github.com/arduino/arduino-cli/pull/1883/files#diff-b01c8ff87205e83918583aff246152ef27d286cccaa14409dddc17bd37104518R146-R150